deploy/osd-delete-backplane-serviceaccounts/10-delete-backplane-serviceaccounts.CronJob.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: osd-delete-backplane-serviceaccounts
+  namespace: backplane
+spec:
+  failedJobsHistoryLimit: 5
+  successfulJobsHistoryLimit: 3
+  concurrencyPolicy: Replace
+  schedule: "*/10 * * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          restartPolicy: Never
+          serviceAccountName: osd-delete-backplane-serviceaccounts
+          containers:
+            - name: osd-delete-backplane-serviceaccounts
+              imagePullPolicy: Always
+              image: image-registry.openshift-image-registry.svc:5000/openshift/cli:latest
+              args:
+                - /bin/bash
+                - -c
+                - |
+                  for sa in $(oc get sa -A -l managed.openshift.io/backplane=true --no-headers | awk '{print $1 ":" $2}');do
```

Comment:
When RBAC is limited to backplane namespaces I expect this needs to use the same regex to look for SA only in those same namespaces.

Reply:
SubjectPermission and filtering to openshift-backplane ns added